### ISSUE TITLE
New Monterey Application Creation Dialog

### ISSUE THREAD

**Issue by adriatic** [monterey-framework/monterey#427]
### Intention

**I'm submitting a feature discussion request**

**Comment by adriatic** [comment]
Jeroen proposes this approach:

![image](https://cloud.githubusercontent.com/assets/2712405/22300046/3e7c736c-e2f4-11e6-8d25-b5ca949d94c3.png)

**Comment by JeroenVinke** [comment]
[this](https://www.screencast.com/t/N4p57FLQlq) is what I have in mind

**Comment by adriatic** [comment]
> this is what I have in mind

Yes, I know (as you just wrote that in Gitter). My article that I want to write is not finished - I just put the first screenshot, to anchor the discussion that should start soon 😄

**Comment by adriatic** [comment]
### Nik's comments and alterations
